eslint.config.mjs

```
@@ -0,0 +1,39 @@
+import globals from 'globals'
```

ESLint

[error] Replace `'globals'` with `"globals";`

(prettier/prettier)